Create PCB

dejanstevanovic opened the pull request.

dejanstevanovic committed:
Added files via upload

dejanstevanovic committed:
Added files via upload

mithro commented:
Umm, you have added all the files to the root directory. You were suppose to modify the existing files.

dejanstevanovic commented:
Yes, I know, but I made mistakes. First I added all files at root and after that I modified only files at board directorium.Can you delete that files, as I see you have right to accept or not accept my modifications.Sorry for mistakes

```
On Wednesday, April 6, 2016 1:59 PM, Tim Ansell <notifications@github.com> wrote:
```

 Umm, you have added all the files to the root directory. You were suppose to modify the existing files.—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
